prow/gangway/example/main.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+
+	pb "k8s.io/test-infra/prow/gangway"
+	gangwayGoogleClient "k8s.io/test-infra/prow/gangway/google"
+)
+
+var (
+	addr      = flag.String("addr", "127.0.0.1:50051", "Address of grpc server.")
+	apiKey    = flag.String("apiKey", "", "API key.")
```

Comment:
so far the convention for command line args in this code base is dash separated instead of camel case, for example https://github.com/kubernetes/test-infra/blob/6a010f363193c6b03267666f80286567aa962762/prow/flagutil/github.go#L132, might want to consider keep consistency